src/main/java/net/ccbluex/liquidbounce/features/module/modules/world/ChestStealer.kt

```diff
@@ -198,6 +210,17 @@ object ChestStealer : Module("ChestStealer", Category.WORLD, hideModule = false)
         }
     }
 
+    private fun getCords(slot: Int): IntArray {
+        val x = slot % 9
+        val y = slot / 9
+        return intArrayOf(x,y)
+
+    }
+    private fun getDistance(from:IntArray,to:IntArray): Int {
+       val distance = (from[0]-to[0])*(from[0]-to[0]) + (from[1]-to[1])*(from[1]-to[1])
+        return distance
+    }
```

Comment:
In Java those int arrays are a good option, but luckily Kotlin gives you `Pair<Int, Int>` which is a better fit here.

The name `getDistance` is misleading. `getDistanceBetweenSlotsSquared` would be less confusing and might prevent bugs.

Reply:
true